core/src/main/java/de/danoeh/antennapod/core/preferences/SleepTimerPreferences.java

```diff
@@ -69,4 +74,19 @@ public static boolean autoEnable() {
         return prefs.getBoolean(PREF_AUTO_ENABLE, false);
     }
 
+    public static void setAutoEnableFrom(int hourOfDay) {
+        prefs.edit().putInt(PREF_AUTO_ENABLE_FROM, hourOfDay).apply();
+    }
+
+    public static int autoEnableFrom() {
+        return prefs.getInt(PREF_AUTO_ENABLE_FROM, DEFAULT_AUTO_ENABLE_FROM);
+    }
+
+    public static void setAutoEnableTo(int hourOfDay) {
+        prefs.edit().putInt(PREF_AUTO_ENABLE_TO, hourOfDay).apply();
+    }
+
+    public static int autoEnableTimeTo() {
```

Comment:
This method has the word "time" in its name, the others don't. I think it should be consistent (one way or the other)

Reply:
I remove the "time".